# this patch avoids downloading astropy-helpers by astropy installer,
# failing in EasyBuild. Instead, we download astropy-helpers as a part
# of the bundle.
#
# - 14.3.2019, J. Dvoracek (Institute of Physics | Czech Academy of Sciences, www.fzu.cz
#
--- ./setup.cfg.orig    2019-03-13 18:46:28.060712000 +0100
+++ ./setup.cfg 2019-03-13 18:46:36.431744000 +0100
@@ -26,7 +26,7 @@
 bitmap = static/wininst_background.bmp
 
 [ah_bootstrap]
-auto_use = True
+auto_use = False
 
 [flake8]
 exclude = extern,*parsetab.py,*lextab.py
